plugins/ProfessionalServices/changes.json

```
@@ -1,4 +1,11 @@
 [
+  {
+    "title": "Funnels - Enhanced Analysis and Comparison Capabilities",
+    "description": "Funnels now offer an improved visualization and data table that provide a clear breakdown of proceeded visits, new entries, exits, and skipped steps at each stage, making it easier to identify areas for optimization. The comparison functionality is improved to allow for a more useful analysis of funnel performance across different segments and date ranges.",
```

We should use British spelling..
```suggestion
    "description": "Funnels now offer an improved visualisation and data table that provide a clear breakdown of proceeded visits, new entries, exits, and skipped steps at each stage, making it easier to identify areas for optimisation. The comparison functionality is improved to allow for a more useful analysis of funnel performance across different segments and date ranges.",
```
Just out of curiosity, has the text been consulted with our technical writer?

---

It looks like Stan provided some updated wording that uses UK spelling. He did consult with Gillian on the wording.

---

@snake14 @michalkleiner Updated text